example/android/app/build.gradle

```diff
@@ -56,7 +56,7 @@ flutter {
 dependencies {
     testImplementation 'junit:junit:4.13'
     //implementation fileTree('libs')
-    implementation 'com.clevertap.android:clevertap-android-sdk:4.6.3'
+    //implementation 'com.clevertap.android:clevertap-android-sdk:4.6.6'
     implementation "com.clevertap.android:push-templates:1.0.1"
```

Comment:
@shivamsharma2710 Update push templates to latest version please?

Reply:
This is example's build.gradle and few things are below,

- The latest Push template is v1.0.7.
- We've added android 13 support in v1.0.6. 

So, should I use version prior to that(i.e. v1.0.5)?

Refer changeLog for reference -> https://github.com/CleverTap/clevertap-android-sdk/blob/master/docs/CTPUSHTEMPLATESCHANGELOG.md

Reply:
Yes use 1.0.5 @shivamsharma2710

Reply:
This is done.

Reply:
Can we close this MR?